WcaOnRails/spec/factories/users.rb

```diff
@@ -47,7 +47,10 @@
     end
 
     trait :wca_id do
-      wca_id { FactoryGirl.create(:person, name: name).id }
+      transient do
```

Comment:
Just curious, what's this `transient do` from? Is it a FactoryGirl thing? A quick google didn't lead me anywhere...